### DISCUSSION TITLE
Add ffmpeg support

### DISCUSSION THREAD

**Discussion by guitoof** [supabase/supabase#27280] [Feature Requests]
[ffmpeg](https://ffmpeg.org/ffmpeg.html) is an amazing media converter.

Currently it is impossible to use the JS library fluent-ffmpeg on Supabase's Edge Functions because the system running the Deno function does not have ffmpeg installed.
Being able to use edge functions to manipulate and convert media files uploaded to the supabase storage would be a great addition.

It seems to be possible with [firebase's Cloud Functions](https://fireship.io/lessons/ffmpeg-useful-techniques/#:~:text=Using%20FFmpeg%20in%20Firebase%20Cloud,js.)

Related to issue: [3773](https://github.com/orgs/supabase/discussions/3773)

**Comment by tomislavhren** [comment]
Do you happen to have any updates on this?
If not, do you have any recommendations on how to do basic audio processing in the Supabase Edge function?

**Comment by TomFrankly** [comment]
Sharing this in case it's helpful – there's an FFMPEG-specific API called Rendi: https://www.rendi.dev/

I'm not affiliated, just thought I'd share it as I landed here hoping to see ffmpeg support for Supabase functions and hit a dead end just as the last person to reply did.

**Reply by peternaf** [reply]
Rendi's founder here :raised_hand_with_fingers_splayed: 
Thanks @TomFrankly for mentioning us!
We built Rendi exactly to support this type of use case based on our own similar frustration.
For anyone coming through this git issue, here is a 35$ coupon code: SUPARENDI

**Comment by joewinke** [comment]
Its been a year, any update? This would enable tons of media potential.

**Comment by galex** [comment]
This is a must-have feature, I can't believe Firebase has it but not Supabase 😨

**Comment by siddiqss** [comment]
I was looking for video to audio conversion with ffmpeg. My main infrastructure uses Supabase but I was disappointed to see that Supabase functions don't support ffmpeg.

**Comment by yerffejytnac** [comment]
Add me to the list. Disappointed that nobody from supabase is even acknowledging these comments 😣

**Comment by aantti** [comment]
@laktek @kallebysantos

**Reply by kallebysantos** [reply]
Hey guys💚
Even the ffmpeg is not installed within the edge-runtime container image. 

I believe that is possible to run it using wasm modules, like in this [image-magick example](https://supabase.com/docs/guides/functions/examples/image-manipulation)

**Comment by huy9k** [comment]
Any tips?
